generic_config_updater/change_applier.py

```diff
@@ -0,0 +1,141 @@
+import copy
+import json
+import jsondiff
+import importlib
+import os
+import tempfile
+from collections import defaultdict
+from swsscommon.swsscommon import ConfigDBConnector
+from .gu_common import genericUpdaterLogging
+
+
+UPDATER_CONF_FILE = "/etc/sonic/generic_config_updater.conf"
+updater_data = None
```

Comment:
global vars are only used in one class, better to move as class member.

Add some comment, especiall what is `updater_data`?